_internal/utils/sourceme.R

```diff
@@ -206,9 +61,19 @@ restore_and_run <- function(app_name, username = "insightsengineering", repo = "
   #'
   #' @param app_directory The directory path of the Shiny app.
   #' @param ... Additional arguments passed to `shiny::runApp()`.
-  load_and_run_app <- function(app_directory, ...) {
-    renv::load(app_directory)
-    renv::restore(clean = TRUE, prompt = FALSE)
+  load_and_run_app <- function(app_directory, package_repo, ...) {
+    renv::dependencies()$Package |>
```

Comment:
shall we use `path` argument here? It's a little cryptic to me where `renv` is looking for dependencies